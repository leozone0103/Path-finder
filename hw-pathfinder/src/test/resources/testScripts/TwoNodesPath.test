# Test findPath between two nodes
CreateGraph graph
AddNode graph A
AddNode graph B

AddEdge graph A B 5.4321

FindPath graph A B