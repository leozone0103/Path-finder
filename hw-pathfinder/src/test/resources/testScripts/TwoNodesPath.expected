# Test findPath between two nodes
created graph graph
added node A to graph
added node B to graph

added edge 5.432 from A to B in graph

path from A to B:
A to B with weight 5.432
total cost: 5.432